src/Http/Livewire/Inbox.php

```diff
@@ -24,6 +26,21 @@ public function selectEmail($id)
 
     public function render()
     {
-        return view('mailthief::livewire.inbox');
+        $emails = MailThief::latest()
+            ->limit(self::EMAILS_PAGE_LIMIT)
+            ->paginate(
+                self::EMAILS_PAGE_LIMIT,
+                ['id','from','subject','attachments','created_at']
+            )
+            ->onEachSide(1);
+
+        return view('mailthief::livewire.inbox', [
+            'emails' => $emails,
```

Comment:
public property `emails` is replaced with instant template variable because property doen't works well when pagination loads.